### DISCUSSION TITLE
4K 3840x2160 16:9 Resolution (Testing and Feedback, 4K Gaming Future?)

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#910] [Q&A] [answered]
I successful tested my full 10 tile Fog of War sight radius (big circle) with 4K 3840x2160 16:9 Resolution (It works on my big monitor)
https://www.deviantart.com/withinamnesia/art/4K-3840x2160-16-9-Resolution-916739972

Resolutions.txt
"# Screen and icon resolution settings

menu_frame_width=640
menu_frame_height=480

icon_size=128

required_width=3840
required_height=2160

virtual_height=2160
virtual_dpi=91.754738

#3840 	× 	2160"

I am using the 4K / x4 times bigger/more/greater pixel assets (even though they look like the old ones but x4 enlarged for prototyping).
So far this 4K / x4 embiggened/enlarged Resolutions.txt works and I am wondering if in ~2-3 years (Hopeful initial game launch date 2024-2025) from now if most people / online gaming community will have at least a 4K monitor that will fit the required display pixel width (3840) and height (2160)? Wikipedia Resolutions: https://en.wikipedia.org/wiki/List_of_common_resolutions

Yet I wonder if things will change anytime soon / in the next 2-3 years (2024-2025)?:
(~1-20[~11]%) 2014: https://linustechtips.com/topic/190920-how-many-people-actually-have-4k-monitors-or-tvs/
(~20-50[~35]%) 2016: https://phantompilots.com/threads/poll-how-many-people-actually-have-a-4k-screen.87485/
(35-55[~45]%)2017-2019: https://www.statista.com/statistics/818419/world-tv-market-share-by-type/
(~2/3 run games at 1080 but 2/3 can run ~4K, Not many 4K games on steam) 2021: https://www.reddit.com/r/Steam/comments/q38oiy/its_2021_we_all_have_gigantic_4k_monitors_why/ + https://store.steampowered.com/hwsurvey/Steam-Hardware-Software-Survey-Welcome-to-Steam
(~31, 38, 44%[~1.2 times growth every year ~31% [2019], ~37.2% [2020], ~44.64% [2021], ~53.568% [2022], ~64.2816%[2023], ~77.13792% [2024], 92.565504% [2025]) Article data 2019, 2020, 2021: https://www.statista.com/statistics/1247334/4k-ultra-hdtv-us-household-penetration/
(~x3.5 4K Market Growth from 2020 to 2026; ~1.23 times 4K projected market growth every year from 2020-2026) 2020-2026: [https://www.mordorintelligence.com/industry-reports/global-4k-display-resolution-market-industry](https://www.mordorintelligence.com/industry-reports/global-4k-display-resolution-market-industry/)

I think it should pay off, this gambit of requiring 4k Monitors to run my game in the mid 2020's. I remember older 90's and 2000's games where they used current / most popular graphics at the time (now extremely dated looking graphics) and they later suffered severe longevity problems and had a very rough time trying to get remasters or improve the graphics (some like Diablo II Resurrected needed complete graphics overhauls from scratch for a later remastered version).

I think 4K 3840x2160 16:9 Resolution should be enough detail to hold up to / last / stand the test of time (if the gameplay holds up / the game is worth playing). Sure Flare games are not built for a full 3D game world but I think past 4K in like 8K or 16K that the human eye just can't really see that much of a difference. So 4K seems to be the 'sweet-spot' for high noticeable detail but without needing a ton of pixels that may or may not even be noticeable.

The Flare series of games might be one of the last 2.5D pre-rendered line of full / grand scale PC games. As in grand scale games meant for PC / Console / not mobile 'mini games'. That is only if commercial versions of games built with Flare can get successfully released and supported by the creators. So I hope that this 'bridge of technology' can hold up some of the great aspects of classic computer games (while being open source and forever free for the creative world) with also hopefully retaining the technological / graphical / gameplay relevance for the future generations of gaming to be worth playing.

So I think that if I bite the bullet and push for required 4K resolution at launch in will be worth it; and also it might be a bit rocky in the middle 2020's but it will get better as the years march onwards. Yet I hope over time the majority of people in the late 2020's and early 2030's should be able to run the game with 4K resolution. I have a slowly strengthening and further reinforced belief in the middle to late 2020's being able to properly / majorly host 4K games. With the major world wide adoption of 4K resolution monitor demands fueled by the march of ever improving graphical fidelity / immersion as the world continues to hunger for greater and greater storytelling and gaming experiences throughout the decades.

Does that make any sense? What are your thoughts? What resolution monitors do you rock? What plans do you have on upgrading in the future? What do you think about the future of resolutions? Is 1080p fading away or is it going stay on top for a decade? When will 4K gaming reach 50+% market share? What makes sense to you? :-)

**Comment by dorkster** [comment, marked as answer]
Personally, I'll probably be sticking with 1080p for quite a while. My primary 28-inch monitor is 1080p, and I feel like I would have to start increasing my OS's interface scaling if the pixel density was any higher. I have a 13-inch laptop that's also 1080p, and I have to scale to *at least* 125% for things to be comfortably legible.

According to the [Steam Hardware Survey](https://store.steampowered.com/hwsurvey/Steam-Hardware-Software-Survey-Welcome-to-Steam), the most popular (67% of players) resolution is still 1080p. Even if people are looking to upgrade their hardware, I bet most would focus on increasing the performance and fidelity of games running at 1080p over an increase in resolution.

That said, 4K is probably the ideal resolution to target for a 2D game at the moment. If you lower the values in `required_width` and `required_height`, people could still play your game on smaller screens with everything looking crisp. Flare would render the game at 4K and "super sample" it down to fit.

**Reply by WithinAmnesia** [reply]
Okay so there are reasonable options for most people around the world to run the 4K game with ~1080 displays?

I am hoping to get the 4K art assets with the 10 tile Fog of War sight radius (big circle) to work on 1080p. Yet I am not 100% sure if that will work properly. Ideally I can make a game that has an eye toward the future / built to last with also being understanding / accommodating for most people today / able to run on 1080p resolution monitors.

My main worry with developing a 2.5D game in the 2020's is trying to get the graphics with enough fidelity so that it can last quite well into the next decade(s). My main focus is gameplay / game experience but it is hard to argue against that the second focus should be graphical fidelity / longevity. I would put sound design third and monetization 4th or 5th for too many game studios got ruined from wanting to money over making something amazing for the decades to come. People die and you can't take your money with you but you can leave behind some kick ass games hopefully XD.

I am quite worried with sticking to 4K that it might be too demanding. Yet I think that I have to stick to 4K because, well look at the original D2 and Baldur's Gate / all of the 90's and 2000's games. It is hard to say those graphics (not gameplay; that is different) are worth experiencing for most people in the 2020's let alone the 2030's.

I want to make something that stays ~'beautiful' / worth playing for most of the gaming community hoping for a decade or two at least. As a result of that desire for overall game experience longevity I have to get the graphics to last a long long time (hopefully decades). I think and I feel that 4K is required in order to get the best shot / opportunity to make something (hopefully) beautiful for most people to experience that can stand the test of time and still be worth playing.

I don't think Flare is the same hardware demand as a full 4K 3D massive multiplayer game with 10+ kilometer draw distance and full physics simulation / mass destruction / voxel features. It is just I am hoping that I can take a bit of graphics push (4K) today to hopefully keep the game looking 'evergreen' / staying beautiful / worth playing in the future decades.

I am in the process of a move right now and I will need to eventually move my P.C.s to the next set up. Yet in the meant time I have my 4K 3840x2160 16:9 Resolution monitor to work with. I have (a) 1080p laptop(s) / monitors and I do want to stress test / hopefully bring compatibility to most reasonable P.C. set ups around the world for my game. So that pretty much everyone interested around the world can get a fair chance to play the game (hopefully).

I think it would be very dumb / quite stupid of myself to expect people to have overkill P.C.s and monitors to run a game made for most people. Then to also expect that most people will also play the beta / demo and then even pay to play the full version of it when the game only runs on like 1-10% of most people's gaming set up. I don't want to make a [hardware] Crisis type game, I want to make a hardware reasonable type game XD.

I am hoping that I can have most people able to reasonably play the game / have the opportunity to play the game with current popular hardware / screen resolutions but also to have the support for future generations of gaming set ups. I just hope that I can figure out how to balance the needs of the present and the demands of the future all at once. I have a real shot at making a great game here and I am trying to be as careful as I can for an eventual launch in the (hopefully) middle 2020's and for the overall longevity of the experience offered to the gaming community this decade and the of the coming future decades.
